docs/guides/migration-guides.md

```diff
@@ -6,10 +6,10 @@ These cover all the major Uppy versions and how to migrate to them.
 
 - End-of-Life versions of Node.js are no longer supported (use latest 18.x LTS,
   20.x LTS, or 22.x current).
-- Setting the `corsOrigin` option is now required. You should define the list of
-  origins you expect your app to be served from, otherwise it can be
-  impersonated from a different origin you don’t control. Set it to `true` if
-  you don’t care about impersonating.
+- Setting the `corsOrigin` (`COMPANION_CLIENT_ORIGINS`) option is now required.
+  You should define the list of origins you expect your app to be served from,
+  otherwise it can be impersonated from a different origin you don’t control.
+  Set it to `true` if you don’t care about impersonating.
```

Comment:
I feel like "impersonating" is unclear and it's not implicit that your API keys can be abused. Would be better to rewrite to illustrate the severity.

Reply:
feel free to suggest a rewrite

Reply:
It was there before this PR, it seems fine to merge as is